SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -78,11 +78,25 @@ def add? {w : Nat} (x y : BitVec w) : IntW w :=
 theorem add?_eq : LLVM.add? a b  = .some (BitVec.add a b) := rfl
 
 @[simp_llvm_option]
-def add {w : Nat} (x y : IntW w) : IntW w := do
+def add {w : Nat} (x y : IntW w) (nswnuw : Bool × Bool := ⟨ false , false ⟩ ) : IntW w := do
```

Comment:
```suggestion
def add {w : Nat} (x y : IntW w) (nswnuw : Bool × Bool := ⟨ false , false ⟩ ) : IntW w := do
```

Maybe it's just me, but I find putting the options into a `structure` to be cleaner. But IIRC, you had this argument with @goens who ruled in favour of not having a `structure`?

Reply:
Where was this discussion? I also feel a structure is cleaner